test/run-e2e-tests.sh

```diff
@@ -25,9 +25,9 @@ SKIP_INSTALL=${SKIP_INSTALL:-}
 if [ -n "$KIND_E2E" ]; then
     # If we did not set SKIP_INSTALL
     if [ -z "$SKIP_INSTALL" ]; then
-        K8S_VERSION=${KUBERNETES_VERSION:-v1.25.0}
```

Comment:
@JaneLiuL also has this change in her 1.26 ga bump: https://github.com/kubernetes-sigs/descheduler/pull/1015/files#diff-b99dcd844076be461a8082515888ea0300eff8977140a8df03cc91a2c829a6c6, fyi one of you may need to rebase